<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>487</width>
    <height>620</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="13" column="0">
    <widget class="QLabel" name="label_optimization_type">
     <property name="text">
      <string>Optimization type</string>
     </property>
    </widget>
   </item>
   <item row="10" column="0">
    <widget class="QLabel" name="label_solution_type">
     <property name="text">
      <string>Solution type</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_iterations">
     <property name="text">
      <string>Iterations</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_penalty_factor">
     <property name="text">
      <string>Penalty factor</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QSpinBox" name="spinbox_cpus"/>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="label_volume_factor">
     <property name="text">
      <string>Volume factor</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QTextEdit" name="textedit_fea_message"/>
   </item>
   <item row="14" column="0">
    <widget class="QComboBox" name="combobox_optimization_type">
     <item>
      <property name="text">
       <string>separate</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="11" column="0">
    <widget class="QComboBox" name="combobox_solution_type">
     <item>
      <property name="text">
       <string>static</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>heat</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_cpus">
     <property name="text">
      <string>CPUs</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QSpinBox" name="spinbox_iterations">
     <property name="maximum">
      <number>1000</number>
     </property>
     <property name="value">
      <number>100</number>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QDoubleSpinBox" name="doublespinbox_penalty_factor">
     <property name="value">
      <double>3.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label_fea_message">
     <property name="text">
      <string>Message from FEA</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QDoubleSpinBox" name="doublespinbox_volume_factor">
     <property name="value">
      <double>0.400000000000000</double>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
